### ISSUE TITLE
Settings style not saved

### ISSUE THREAD

**Issue by ddnexus** [vsch/idea-multimarkdown#585]
I created an empty project with just one .md document in it... Then I added some custom style and exported the settings. Every time that I close the project, when I re-open it I have to re-import the styles, since the additions have not been saved. Is it a possible bug or I am just missing something?

**Comment by vsch** [comment]
@ddnexus, if by custom styles you are referring to Rendering profiles  and by styles you mean custom CSS then yes. 2018 has a bug, #582 

> You need to be aware that ALL 2018 IDEs have a bug when saving rendering profile settings for the plugin. They will loose all text fields if they contain whitespaces (ie. spaces or EOL). Which basically means all custom css and html content. See https://youtrack.jetbrains.com/issue/IDEA-184506, if you are affected, feel free to vote it up and/or complain to JetBrains. **I provided a patch Feb 25th, however it has not made it into the release.**

**Specifically, I did the job of finding the bug, reporting it and finally took the time to fix it and provide a patch. At this point I think it is out of my hands and up to the users to help get JetBrains' attention to have the one line fix incorporated into the release.**

**Comment by vsch** [comment]
@ddnexus, for myself I cannot use an IDE which looses these settings. I have many of my builds depend on customized Export of Markdown files.

I wrote a batch script to patch all the IDEs that I use with the fix. It is a PITA to go through the procedure every time an IDE is updated. I had to create a self issued signing certificate and sign the modified apps with it so that they will be treated as official apps for security purposes on my system.

I have had no feedback from JetBrains on when the patch will be incorporated into the release. I find this a serious gap in support since everything that I could do to make it happen I already did, other than start screaming loudly which I am not sure will help. 

My experience with quality of support varies greatly depending on your contact person and issue. Especially when I raise issues that are exposed by my plugin. **Third party plugin Issues** appear to get lower priority with some people at JetBrains while others were a pleasure to deal with, leading to an amazing experience in promptness and willingness to help. Luck of the draw.

I have developed mixed emotions regarding reporting issues. Each takes an enormous amount of time to isolate, prepare documentations and report. Often leading to questionable ROI for the effort. I only do it when using the IDE for me becomes impossible and in many of these cases wound up fixing the bug by either dealing with it in my plugin or by fixing it in the IDE.
